manager.go

```diff
@@ -95,6 +96,35 @@ func (m *Manager) LoadConfig(cfgPath string) error {
 	return nil
 }
 
+var ErrMissingCloudCreds = errors.New("can't create NetAppender without cloud creds")
+
+// create and attach a NetAppender.
+func (m *Manager) attachNetAppender() error {
+	if m.conn == nil {
+		return errors.New("NetAppender requires non-null conn")
+	}
+	if m.netAppender != nil {
+		return errors.New("Manager already has non-nil netAppender")
+	}
+	if m.cloudAddr == "" || m.partID == "" || m.cloudSecret == "" {
+		return ErrMissingCloudCreds
+	}
+	netAppender, err := logging.NewNetAppender(
+		&logging.CloudConfig{
+			AppAddress: m.cloudAddr,
+			ID:         m.partID,
+			Secret:     m.cloudSecret,
+		},
```

Comment:
Probably harmless (and/or good practice), but normally this should never be used, as we're passing a connection directly, right? I'm a little uncertain if we DO pass creds here, if/when it might use them and re-dial its own separate connection. Do we want that? (Genuinely asking!)